dub.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "gtk-d",
+    "version": "4.0.0",
```

Comment:
This is also in an other pull request.
Doesn't dub get the version from the version control / git?

Reply:
Using GIT branches as project versions is deprecated: https://dub.pm/package-format-json#version-specs
And I also think that installing a version is a good habit.

Reply:
While using a git branch as the version of a dependency is indeed deprecated. I can't find anything about the way you are using version here in the dub documentation.

Reply:
OK. Do you want me to delete it?

Reply:
Yes.

It does look like dub supports setting the version in dub.json or .sdl, and it takes precedence over the version based on the git tag. Though it's strange it is not documented on code.dlang.org.

By not specifying the version it will always be in sync with git.

Reply:
Ok. I have done it